index.html

```diff
@@ -1634,6 +1788,67 @@ <h2>Dictionary <a class="idlType">RTCQuicStreamAbortInfo</a> Members</h2>
         </section>
       </div>
     </section>
+    <section id="quicsenddatagramresult*">
+      <h3><dfn>QuicSendDatagramdResult</dfn> Dictionary</h3>
+      <p>The <code>QuicStreamReadResult</code> includes information
+      relating to the result returned from <code>sendDatagram</code>.</p>
+      <div>
+        <pre class="idl">interface QuicSendDatagramResult {
+	     readonly attribute bool blocked;
+             readonly attribute Promise unblocked;
+             readonly attribute Promise&lt;bool&gt; acked;
+        };
+        </pre>
+        <section>
+          <h2><a class="idlType">QuicSendDatagramResult</a> attributes</h2>
+          <dl data-link-for="QuicSendDatagramResult" data-dfn-for="QuicSendDatagramResult" class=
+          "attributes">
+            <dt><dfn><code>blocked</code></dfn> of type <span class=
+            "idlAttrType"><a>bool</a></span>.</dt>
+            <dd>
+              <p>Whether or not the datagram was dropped due to flow

```

Comment:
Please only reference congestion control.  I don't believe there's a practical way to implement flow control for datagrams, and I tend to think it's undesirable as well.

Reply:
Good idea.  Done.

Reply:
Actually, I found this in the datagram draft: "In cases in which a DATAGRAM frame is blocked due to connection-level flow control or congestion control, an implementation MAY drop the frame without sending it." (https://tools.ietf.org/html/draft-pauly-quic-datagram-00)

So should we use that phrase ("blocked due to connection-level flow control or congestion control")?